src/components/chat/comment-card.tsx

```diff
@@ -31,22 +59,27 @@ export const CommentCard: React.FC<IProps> = ({ activeNavTab, user, postedCommen
   return (
     <div className={`comment-card selected`} data-testid="comment-card">
       {renderThreadHeader()}
-      { postedComments?.map((comment, idx) => {
+      {postedComments?.map((comment: any, idx) => {
```

Comment:
```suggestion
      {postedComments?.map((comment, idx) => {
```
This should be properly typed without the `any`.